cli/cmd/volume.go

```diff
@@ -202,13 +210,36 @@ var volumeGetCmd = &cobra.Command{
 	},
 }
 
+func volumeOptionJsonHandler(cmd *cobra.Command, volname string, options []string) error {
+	vopt := make(map[string]string)
+	for op,val := range options {
+		if op % 2 == 0 {
+			vopt[val] = options[op+1]
+		}else{
+			continue
+		}
+	}
+	var op = map[string]interface{}{"options": vopt}
```

Comment:
Avoid interface where not required. Try,

var op = map[string]map[string]string

Reply:
Noted.